### DISCUSSION TITLE
Capture output sa html

### DISCUSSION THREAD

**Discussion by lewoudar** [Textualize/rich#2159] [Q&A]
Hi everyone, I'm wondering if I can do something like the following:

```python
from rich.console import Console
console = Console()
with console.capture() as capture:
    console.print("[bold red]Hello[/] World")
str_output = capture.get_html_output()
```

i.e get the HTML output (using `get_html_output` in my dummy example) when capturing the data, it seems like it is impossible right now 🙁

**Comment by willmcgugan** [comment]
See [exporting](https://rich.readthedocs.io/en/latest/console.html#exporting)

**Reply by lewoudar** [reply]
I know but the idea is not to print the output in terminal, just export it at the end in HTML, is not possible? 
Something like this:

```python
console = Console(record=True)
with console.capture() as capture:
    console.print("[bold red]Hello[/] World")  # <= not print in the terminal
str_output = capture.save_html('test.html') # <= here we should have Hello in bold red
```
